zigpy/ota/provider.py

```diff
@@ -535,3 +536,85 @@ async def refresh_firmware_list(self) -> None:
                     self._cache[img.key] = img
 
         self.update_expiration()
+
+
+@attr.s
+class INOVELLIImage:
+    manufacturer_id = attr.ib()
+    image_type = attr.ib()
+    version = attr.ib(default=None)
+    image_size = attr.ib(default=None)
+    url = attr.ib(default=None)
+
+    @classmethod
+    def new(cls, data):
+        mod = data["model"]
+        ver = data["version"]
+        url = data["firmware"]
+
+        res = cls(
+            manufacturer_id=Inovelli.MANUFACTURER_ID, model=mod, version=ver, url=url
+        )
```

Comment:
Similarly, when parsing the JSON object (and the `model` key), you would have to parse the string version into a number:

```python
@classmethod
def new(cls, data, model):
    ver = int(data["version"], 10)
    url = data["firmware"]

    res = cls(
        manufacturer_id=Inovelli.MANUFACTURER_ID, model=model, version=ver, url=url
    )
```